app/build.gradle.kts

```diff
@@ -107,4 +107,5 @@ dependencies {
     androidTestImplementation(Testing.hiltTesting)
     kaptAndroidTest(DaggerHilt.hiltCompiler)
     androidTestImplementation(Testing.testRunner)
+    androidTestImplementation(Testing.testCore)
```

Comment:
I added `androidx.test:core` because of this issue.

https://github.com/android/android-test/issues/196